python/health-check/src/health_check/config/templates/grafana_dashboard/supportconfig_with_logs.template.json

```diff
@@ -70,9 +83,9 @@
         "x": 0,
         "y": 6
       },
-      "id": 109,
+      "id": 71,
       "panels": [],
-      "title": "Memory",
+      "title": "Uyuni Server and Salt",
```

Comment:
Note, same as above - I wanted to remove all `uyuni` occurrences from the Grafana board as well.